### PR TITLE
Porfolio project part 2 from cmauer69

### PR DESCRIPTION
**Pull request by cmauer69**
Mary,
I know I could do better on this project, but that will have to come later.  Right now time is up and it has to be turned in.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  <title>Clifford Mauer's Portfolio</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+ <link rel="stylesheet" href="Styles.css" type="text/css">  
+  <link rel="stylesheet" href="iconstyle.css" type="text/css"> 
+<style>
+  #myScrollspy {
```
Comment (htcMosman) [inline index.html]: You want to get all of your CSS in the stylesheet file and not have any in the HTML

File: index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    <div class="container">
+    <div class="row">
+        <div class="col-sm-4">
+        <img src="images/Snapshot_20140227.JPG" class="img-responsive" alt="Snapshot of Cliff Mauer" width="400" height="200" >
+        </div>
+        <div class="col-sm-6">
+            <div id="name">
+                <h1 style="text-align:center">Clifford J. Mauer</h1></div>
+    <h2>Email: cmauer@mauerenterprises.com</h2>
+    <h2>Phone: (612)730-9965</h2>
```
Comment (htcMosman) [inline index.html]: You could do an email link and one for the phone on the mobile site.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  </div>
+</nav>    
+<div data-spy="scroll" data-target=".navbar" data-offset="12" id="myScrollspy">
+  <div id="section1">
+    <div class="container-fluid">
+      <h1>Skills</h1>
+    <div class="row">
+        <div class="col-sm-3" style="font-size:50%">
+        <ul>
+            <li><span class="icon icon-star"></span> c# programming language</li>
```
Comment (htcMosman) [inline index.html]: I love the icon and text combination, but maybe aim to size them a little better.  The icon feels to big for the text, and while the text seems small, I think that's just because the icon is so big.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+            <li><span class="icon icon-paperplane"></span> Machining</li>
+            <li><span class="icon icon-star"></span> Car repairs</li>
+            <li><span class="icon icon-star"></span> Problem Solving</li>
+            <li><span class="icon icon-star"></span> Supervising</li>
+           <li><span class="icon icon-star"></span> Teaching</li>
+       </ul></div>
+    </div>
+    </div>
+  </div>
+  <div id="section2">
```
Comment (htcMosman) [inline index.html]: Look for a way to space things out a little more in each section.  More of padding or margin at the top/bottom can help give it a little more spacious feel.  You might also add some sort of image for the desktop view and alternate right/left side to help break up the text and keep things interesting.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+        </ul></div>
+       <div class="col-sm-4" style="font-size:50%">
+        <ul>
+            <li>Hampshire Labs Inc.</li>
+            <li>Werner Enterprises Inc.</li>
+            <li>Video Age Inc.</li>
+           </ul></div></div></div></div>
+   <div id="section3">
+    <div class="container-fluid">
+      <h1>Projects</h1>
```
Comment (htcMosman) [inline index.html]: Don't forget you can link to GitHub here and to your web projects from class.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+            </div></li>
+        <li>Hennepin Technical Institute
+          <div id="hennepin">Currently studying new programming languages</div></li>
+      </ul>
+    </div>  
+  </div>
+    </div></div>
+<footer>
+    <ul>
+        <li><span class="icon icon-facebook2"></span> Facebook</li>
```
Comment (htcMosman) [inline index.html]: If you're not on LinkedIn, you should join... if for no other reason than the alumni group and happy hours.  :-)

**Comment by htcMosman** [comment]
Structurally this looks good, but visually I think some spacing and a little more attention to the styling could go a long way.  More images (or even clever use of those icons) could make it more "fun" but aren't really necessary either.
